index.js

```diff
@@ -3,6 +3,7 @@ var app = express()
 
 app.all('*', (req, res) => {
   res.json({
+    service: process.env.SERVICE_NAME || 'Define environment variable SERVICE_NAME',
```

Comment:
Sorry it's taken so long for me to get back to you on this.

I would prefer to omit `SERVICE_NAME` from the response if not provided. This way, it's an enhancement that users can opt into. If they don't need or use a service name parameter, then the warning message would clutter their display needlessly.

```suggestion
    service: process.env.SERVICE_NAME || undefined, // Keys with value `undefined` are omitted during JSON serialization
```

I'd prefer to also add a short line about this to the README.

Thank you for your contribution @Yatufo!